examples/experimental/v1alpha2/grpc-routing/bar-grpcroute.yaml

```diff
@@ -1,23 +0,0 @@
-#$ Used in:
-#$ - site-src/guides/grpc-routing.md
```

Comment:
Looks like this and all other examples removed in this PR need to move to v1 + standard channel, and have corresponding references updated in https://deploy-preview-3361--kubernetes-sigs-gateway-api.netlify.app/guides/grpc-routing/

Reply:
They are actually already there!